tests/test_passwords.py

```diff
@@ -0,0 +1,133 @@
+import json
+from urllib import urlencode
+
+import pytest
+from tornado.httpclient import HTTPError
+
+from fixtures import standard_graph, graph, users, groups, session, permissions  # noqa
+from fixtures import fe_app as app  # noqa
+from grouper import public_key
+from grouper.model_soup import User, Request, AsyncNotification
+from grouper.models.user_password import UserPassword
+from grouper.user_password import add_new_user_password, delete_user_password, PasswordAlreadyExists
+from url_util import url
+
+
+def test_passwords(session, users):
+    user = users['zorkian@a.co']
+
+    add_new_user_password(session, "test", "test_password_please_ignore", user.id)
+    assert len(user.my_passwords()) == 1, "The user should only have a single password"
+    password = user.my_passwords()[0]
+    assert password.name == "test", "The password should have the name we gave it"
+    assert password.password != "test_password_please_ignore", "The password should not be what is passed in"
```

Comment:
if you refactor this out to a constant, we don't have to worry about typos in your checks.